Gemfile

```diff
@@ -1,2 +1,5 @@
 source 'https://rubygems.org'
 gemspec
+
+# needed for now because its not released...
+gem 'motion_print', github: 'OTGApps/motion_print'
```

Comment:
If everyone is good with the functionality i'll go ahead and release a new version of the gem.